packages/nodes-base/nodes/Merge/test/v3/operations.test.ts

```diff
@@ -259,6 +259,142 @@ describe('Test MergeV3, combineBySql operation', () => {
 			country: 'PL',
 		});
 	});
+
+	it('SELECT query, should run executeSelectWithMapedPairedItems and collect pairedItems data', async () => {
```

Comment:
Since you are not testing the implementation please remove executeSelectWithMapedPairedItems from the test description
```suggestion
	describe('SELECT query',()=>{
	it('collects pairedItems data', async () => {
```